frontend/src/pages/analytics/segments/AddUserSegment.tsx

```diff
@@ -47,14 +58,44 @@ const AddUserSegments: React.FC = () => {
       href: `/analytics/${projectId}/app/${appId}/segments`,
     },
     {
-      text: t('breadCrumb.createSegment'),
+      text:
+        actionType === 'edit'
```

Comment:
Change to `ternary(condition, a, b)` from `utils.ts`

Reply:
done